sale_order_min_qty/models/product.py

```diff
@@ -10,7 +10,7 @@ class ProductProduct(models.Model):
 
     sale_min_qty = fields.Float(
         string="Min Sale Qty",
-        default=1,
+        default=0,
```

Comment:
@mourad-ehm No need to `default=0,`, Float default value is `0.0`

Reply:
thanks for review. I removed default value.